recipe/meta.yaml

```
@@ -10,42 +10,55 @@ source:
   # macOS: shasum -a 256 <file>
 
 build:
-  number: 0
+  number: 1
   skip: True  # [win and vc<14]
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
   entry_points:
     - fmpy = fmpy.command_line:main
 
-requirements:
-  build:
-    - python
-    - pip
-    - lxml
-    - numpy
-    - pathlib
-    - pyqt
-    - pywin32  # [win]
-  run:
-    - python
-    - dask
-    - lxml
-    - matplotlib
-    - numpy
-    - pathlib
-    - pyqt
-    - pyqtgraph
-    - pywin32  # [win]
-    - requests
+outputs:
+  - name: fmpy-base
+    script: build_base.bat  # [win]
+    script: build_base.sh  # [not win]
+    requirements:
+      build:
+        - python
+        - pip
+        - lxml
+        - numpy
+        - pathlib
+        - pyqt
+        - pywin32  # [win]
+      run:
+        - python
+        - lxml
+        - numpy
+        - pathlib
+        - pywin32  # [win]
+    test:
+      imports:
+        - fmpy
+        - fmpy.ssp
+        - fmpy.ssp.examples
+        - fmpy.sundials
 
-test:
-  imports:
-    - fmpy
-    - fmpy.cross_check
-    - fmpy.examples
-    - fmpy.gui
-    - fmpy.ssp
-    - fmpy.ssp.examples
-    - fmpy.sundials
+  - name: fmpy
+    requirements:
+      run:
+        - dask
+        - fmpy-base
```

```suggestion
        - {{ pin_subpackage('fmpy-base', max_pin="x.x.x") }}
```

You should pin the subpackage dependency.